e2e/davinci-app/vite.config.ts

```diff
@@ -3,17 +3,6 @@ import { defineConfig } from 'vite';
 
 import { nxViteTsPaths } from '@nx/vite/plugins/nx-tsconfig-paths.plugin';
```

Comment:
non-blocking. but this was probably a missed thing on my end, we can remove this line since we aren't using tsconfig paths.